virttest/qemu_devices/qcontainer.py

```diff
@@ -1542,7 +1574,12 @@ def usb_by_params(self, usb_name, params):
                                      params.get("usb_controller"),
                                      params.get("bus"),
                                      params.get("port"),
-                                     params.get("usbdev_serial"))
+                                     params.get("usbdev_serial"),
+                                     params.get("hostbus"),
```

Comment:
As @yanan-fu said, suggest adding the perfix `usbdev_option_` to these params.

Reply:
Updated, thanks!